### DISCUSSION TITLE
更新日志 - 22.03.6 - 20240510

### DISCUSSION THREAD

**Discussion by jjm2473** [istoreos/istoreos#1559] [Announcements]
# 应用
* QuickStart：
  1. 修复某些DDNS提供商在首页的链接错误 #689
  2. 降低软件源的检测频率，并且在前端页面不活跃时暂停刷新数据 #1541

PS：固件下载慢的话，可以试试从sourceforge手动下载 https://sourceforge.net/projects/istoreos/files/iStoreOS/

**Comment by xpn282** [comment]
在线更新固件慢得出奇，跟百度云盘非会员似的，50k/s。。。
改进一下服务器吧。。。

**Reply by jjm2473** [reply]
发固件第一天才慢吧，周六还慢？

**Reply by jjm2473** [reply]
之后会弄个cdn

**Reply by kangSenjon** [reply]
首页IP地址这里能不能可以选择接口？我现在显示的不是wan口信息，也不能更改

**Comment by bg4evk** [comment]
现在还是很慢

**Comment by maoweicao710** [comment]
巨慢

**Comment by xpn282** [comment]
不只是固件刚发布时慢，其他时候也是慢，顶多200k/s

**Comment by jjm2473** [comment]
在sourceforge增加了一个分流站： https://sourceforge.net/projects/istoreos/files/iStoreOS/

源站下载慢的话可以先用这个分流站

**Comment by mylaracroft** [comment]
爆慢

**Reply by jjm2473** [reply]
试试从sourceforge手动下载 https://sourceforge.net/projects/istoreos/files/iStoreOS/

**Comment by hjc19820909** [comment]
应用过滤这个功能又不能用了。

**Comment by sihan5** [comment]
迅雷打不开了啥情况啊

**Reply by aksehf123** [reply]
我也一样，那天改了下下载地址后就不能使用了，重新安装也没用。。。

**Comment by mochuxi214613** [comment]
x86 更新后进不了管理后台，但是ssh 还有服务都正常

**Reply by fankangsong** [reply]
解决了吗？瑟瑟发抖还没升级

**Comment by deciushu** [comment]
什么时候出一个支持显卡的正式版出来呢

**Comment by aksehf123** [comment]
迅雷更改了下载地址然后按“升级/应用”后就无法使用了，打开页面显示“404 page not found”；尝试删除容器和镜像、删除配置，再重新安装还是无法使用；内置的两个镜像都拉取过显示成功，但实际无法使用。

**Reply by jjm2473** [reply]
修复了，istore里面升级下

**Comment by xiaotiantian1132** [comment]
前端固件升级页面，无法刷写新固件，提示XHR request timed out，有没有后台强制刷写的办法？除了TF卡的方式

**Reply by xiaomeng9597** [reply]
官方下载服务器有问题，访问不稳定
